src/renderer/families/polkadot/Nomination/index.js

```diff
@@ -304,173 +293,177 @@ const Nomination = ({ account }: Props) => {
+            <Box style={{ maxWidth: "65%" }}>
+              <Text ff="Inter|Medium|SemiBold" color="palette.text.shade60" fontSize={4}>
+                <Trans
+                  i18nKey="polkadot.nomination.emptyState.description"
+                  values={{ name: account.currency.name }}
+                />
+              </Text>
+              <Box mt={2}>
+                <LinkWithExternalIcon
+                  label={<Trans i18nKey="polkadot.nomination.emptyState.info" />}
+                  onClick={onLearnMore}
+                />
               </Box>
-            </ToolTip>
-            <Box horizontal>
-              <ToolTip
-                content={
-                  electionOpen ? <Trans i18nKey="polkadot.nomination.electionOpenTooltip" /> : null
-                }
-              >
-                <Button
-                  id={"account-rebond-button"}
-                  disabled={electionOpen}
-                  mr={2}
-                  primary
-                  small
-                  onClick={onRebond}
+            </Box>
+            <Box>
+              {!hasBondedBalance && !hasPendingBondOperation ? (
+                <ToolTip
+                  content={
+                    electionOpen ? (
+                      <Trans i18nKey="polkadot.nomination.electionOpenTooltip" />
+                    ) : null
+                  }
                 >
-                  <Box horizontal flow={1} alignItems="center">
-                    <RebondIcon size={12} />
-                    <Box>
-                      <Trans i18nKey="polkadot.unlockings.rebond" />
+                  <Button primary small disabled={electionOpen} onClick={onEarnRewards}>
+                    <Box horizontal flow={1} alignItems="center">
+                      <ChartLineIcon size={12} />
+                      <Box>
+                        <Trans i18nKey="delegation.title" />
+                      </Box>
                     </Box>
+                  </Button>
+                </ToolTip>
+              ) : (
+                <ToolTip
+                  content={
+                    !nominateEnabled && electionOpen ? (
+                      <Trans i18nKey="polkadot.nomination.electionOpenTooltip" />
+                    ) : !nominateEnabled && hasPendingBondOperation ? (
+                      <Trans i18nKey="polkadot.nomination.hasPendingBondOperation" />
+                    ) : null
+                  }
+                >
+                  <Button primary small disabled={!nominateEnabled} onClick={onNominate}>
+                    <Box horizontal flow={1} alignItems="center">
+                      <NominateIcon size={12} />
+                      <Box>
+                        <Trans i18nKey="polkadot.nomination.nominate" />
+                      </Box>
+                    </Box>
+                  </Button>
+                </ToolTip>
+              )}
+            </Box>
+          </Wrapper>
+        )}
+        {hasUnlockings ? (
```

Comment:
It breaks on Polkadot.

Nominations and Unbondings should be 2 separate Tables.
I might separate the 2 sections into 2 components (included in the AccountBodyHeader.js) after your PR, since we might have a 3rd section for Rewards that would come soon.

This component is too long.


![image](https://user-images.githubusercontent.com/70533374/111323230-f7157a00-8669-11eb-969a-596d42e574e8.png)